Neos.Fusion/Classes/Core/Parser.php

```diff
@@ -609,13 +609,12 @@ protected function parseInclude($include)
             }
             $iterator = new \DirectoryIterator($basePath);
         }
-
         // If iterator is set it means we're doing globbing
         if (isset($iterator)) {
             foreach ($iterator as $fileInfo) {
                 $pathAndFilename = $fileInfo->getPathname();
-                // Only work on .ts2 files
-                if ($fileInfo->getExtension() === 'ts2') {
+                // Only work on .fusion files and .ts2 files. The support for .ts2 is a fallback and will be dropped with 4.0
```

Comment:
Please add the word "deprecated" somewhere in the comment, as this is what we usually search for to find these deprecations in code :)